src/components/Footer/Footer.scss

```
@@ -0,0 +1,153 @@
+.footer {
+  background-color: black;
+  .footerTop {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 3rem 0 3rem;
```

```suggestion
    margin: 0 3rem;
```

이렇게 축약형으로 작성하면 가독성이 더 좋을 것 같군요!

---

넵 수정 완료했습니다.